impl/src/test/java/org/ehcache/impl/serialization/CompactPersistentJavaSerializerTest.java

```diff
@@ -30,6 +31,7 @@
 /**
  * CompactPersistentJavaSerializerTest
  */
+@Ignore
```

Comment:
Ignore this change as this whole class has been removed in https://github.com/ehcache/ehcache3/pull/1435